[0m17:57:27.415005 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000219E429AE90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000219E42FE810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000219E42FE390>]}


============================== 17:57:27.420015 | d4ce095e-aed6-4e92-8247-5f1f72517ca8 ==============================
[0m17:57:27.420015 [info ] [MainThread]: Running with dbt=1.10.3
[0m17:57:27.421019 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': 'C:\\Users\\SarigaMadathil\\.dbt', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'static_parser': 'True', 'introspect': 'True', 'invocation_command': 'dbt init marketing_funnel_analytics', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m17:57:27.477028 [debug] [MainThread]: Starter project path: c:\Users\SarigaMadathil\OneDrive - ROKO Labs LLC\Projects\Growth Analytics\.venv\Lib\site-packages\dbt\include\starter_project
[0m17:57:27.617999 [info ] [MainThread]: 
Your new dbt project "marketing_funnel_analytics" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:57:27.619313 [info ] [MainThread]: Setting up your profile.
[0m17:57:50.130776 [debug] [MainThread]: Spark adapter: Setting pyhive.hive logging to ERROR
[0m17:57:50.131778 [debug] [MainThread]: Spark adapter: Setting thrift.transport logging to ERROR
[0m17:57:50.131778 [debug] [MainThread]: Spark adapter: Setting thrift.protocol logging to ERROR
[0m18:03:05.825824 [info ] [MainThread]: Profile marketing_funnel_analytics written to C:\Users\SarigaMadathil\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m18:03:05.826829 [debug] [MainThread]: Command `dbt init` succeeded at 18:03:05.826829 after 338.51 seconds
[0m18:03:05.827831 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000219E52F8B90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000219DD5B2290>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000219E42A0450>]}
[0m18:03:05.828831 [debug] [MainThread]: Flushing usage events
[0m18:03:06.466847 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m15:00:13.479508 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016FEB59F690>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016FEB99E1D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016FEB99D7D0>]}


============================== 15:00:13.484561 | dac6d6f1-182d-41fa-9c12-4b7ed74f8aae ==============================
[0m15:00:13.484561 [info ] [MainThread]: Running with dbt=1.10.3
[0m15:00:13.485456 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\SarigaMadathil\\.dbt', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptionsV2(error=[], warn=[], silence=[])', 'invocation_command': 'dbt run', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:00:13.485456 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\SarigaMadathil\OneDrive - ROKO Labs LLC\Projects\Growth Analytics\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m15:00:13.487465 [debug] [MainThread]: Command `dbt run` failed at 15:00:13.487465 after 0.11 seconds
[0m15:00:13.488463 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016FE4F781D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016FEB9F75D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016FEB9F6C90>]}
[0m15:00:13.491047 [debug] [MainThread]: Flushing usage events
[0m15:00:14.105040 [debug] [MainThread]: An error was encountered while trying to flush usage events
